test/integration/processing.test.js

```diff
@@ -0,0 +1,268 @@
+const mockSendBatchMessages = jest.fn()
+const mockSendEvent = jest.fn()
+jest.mock('ffc-messaging', () => {
+  return {
+    MessageBatchSender: jest.fn().mockImplementation(() => {
+      return {
+        sendBatchMessages: mockSendBatchMessages,
+        closeConnection: jest.fn()
+      }
+    })
+  }
+})
+jest.mock('ffc-pay-event-publisher', () => {
+  return {
+    PublishEvent: jest.fn().mockImplementation(() => {
+      return {
+        sendEvent: mockSendEvent
+      }
+    }),
+    PublishEventBatch: jest.fn().mockImplementation(() => {
+      return {
+        sendEvents: jest.fn()
+      }
+    })
+  }
+})
+const pollInbound = require('../../app/processing/poll-inbound')
+const { BlobServiceClient } = require('@azure/storage-blob')
+const db = require('../../app/data')
+const config = require('../../app/config/storage')
```

Comment:
dbl checking these var names

Reply:
`db` is fine and consistent with the code.  Renamed `config` to `storageConfig` to be clearer what it is.